src/components/OverviewMap.js

```diff
@@ -131,7 +131,7 @@ class OverviewMap extends React.Component {
           default:
           // TODO: Implement support for other geometries too (markers, square, circle)
             contents.push(
-              <GeoJSON data={geojson} key={JSON.stringify(geojson)}>{this.getPopupContent(hearing, geojson)}</GeoJSON>
+              <GeoJSON data={geojson} key={Math.random()}>{this.getPopupContent(hearing, geojson)}</GeoJSON>
```

Comment:
Math.random does not return a unique number, although changes of duplicates is very small. Could still mix this random value with something to make it unique, date string or or the hearing id upper in the scope 🤔

Reply:
Would most likely to fix this for other switch cases too

Reply:
Well this is now implemented in smarter way, it still has a small chance of having duplicate key because a single hearing can have multiple markers but complicating this further would be close to pointless as there are no dangers associated with the map rendering incorrectly once in a billion tries.